envoy/stream_info/filter_state.h

```diff
@@ -66,6 +93,16 @@ class FilterState {
     virtual absl::optional<std::string> serializeAsString() const { return absl::nullopt; }
   };
 
+  struct FilterObject {
+    std::shared_ptr<Object> data_;
+    StateType state_type_;
+    StreamSharing stream_sharing_{StreamSharing::None};
+    std::string name_{""};
```

Comment:
The initialization can just be `{}` instead of `{""}`, but empty is the default so it can be omitted entirely.

Reply:
Done.